index.html

```diff
@@ -27,42 +23,79 @@
     </div>
     <div class="main-body">
         <div class="main-logo">
-            <img src="./images/icon4.svg" width="218" ; height="55" ; />
+            <img src="./images/icon4.svg" width="218"  height="55" alt="" />
         </div>
         <div class="main-icon">
             <div class="sub-menu-icon">
-                <div><a href="#"><img src="./images/icon1.svg" width="23" height="23" /></a> </div>
+                <div><a href="#"><img src="./images/icon1.svg" width="23" height="23" alt="" /></a> </div>
                 &nbsp;
                 <div> <a href="#">mint</a></div>
             </div>
             <div class="sub-menu-icon">
-                <div><a href="#"> <img src="./images/icon2.svg" width="23" height="23" /></a></div>
+                <div><a href="#"> <img src="./images/icon2.svg" width="23" height="23" alt="" /></a></div>
                 &nbsp;
                 <div> <a href="#">quickbooks</a></div>
             </div>
             <div class="sub-menu-icon">
-                <div><a href="#"> <img src="./images/icon3.svg" width="23" height="23" /></a></div>
+                <div><a href="#"> <img src="./images/icon3.svg" width="23" height="23" alt=""/></a></div>
```

Comment:
I think the alt should have some content; if it doesn't have any then it is useless for the visually impaired people. Also, your icons are untidy.
```suggestion
                <div><a href="#"> <img src="./images/icon2.svg" width="23" height="23" alt="turbotax logo"/></a></div>
```